src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -82,29 +113,54 @@ const ActiveState = ({
                                 translations={translations}
                             />
                         </li>
-                    ) : null;
-                case 'file_version':
-                    return (
-                        <li key={type + id} className="bcs-version-item">
+                    </CellMeasurer>
+                ) : null;
+            case 'file_version':
+                return (
+                    <CellMeasurer cache={this.cache} columnIndex={0} key={key} parent={parent} rowIndex={index}>
+                        <li key={type + id} className="bcs-version-item" style={style}>
                             {versions ? (
                                 <CollapsedVersion {...item} onInfo={onVersionInfo} />
                             ) : (
                                 <Version {...item} onInfo={onVersionInfo} />
                             )}
                         </li>
-                    );
-                case 'keywords':
-                    return (
-                        <li key={type + id} className="bcs-keywords-item">
+                    </CellMeasurer>
+                );
+            case 'keywords':
+                return (
+                    <CellMeasurer cache={this.cache} columnIndex={0} key={key} parent={parent} rowIndex={index}>
```

Comment:
It looks like all four uses of `CellMeasurer` have the same prop values. Should it be extracted to its own simple component?